rllib/agents/trainer.py

```diff
@@ -419,7 +419,8 @@ def __init__(self, config=None, env=None, logger_creator=None):
         config = config or {}
 
         if tf and config.get("eager"):
-            tf.enable_eager_execution()
+            if not tf.executing_eagerly():
+                tf.enable_eager_execution()
```

Comment:
This is idempotent so you don't need to check it.

Reply:
I do remember this broke some test recently. Forgot, which one and in which situation.